localbuild/meta.yaml

```diff
@@ -87,6 +87,9 @@ requirements:
     - pycountry
     - websocket-client
     - libtiff <4.5.0
+    - Flask-WTF
+    - wtforms[email]
```

Comment:
do we have this already on conda-forge? can't find it by mamba search directly

We can add it if it is missing, when it is on pypi we can use grayskull and add it to the staged_recipes in a PR https://github.com/conda-forge/staged-recipes When accepted it gives a wtforms[email]-feedstock repo and a package.

Reply:
The reason I imported `wtforms[email]` is because of a validation error that occurs in `wtforms` latest version. However, `email_validator` also [solving this error](https://stackoverflow.com/a/65056202/13738547), so there's no need to worry about it here. Sorry for any confusion it may have caused.